app/helpers/physical_switch_helper/textual_summary.rb

```diff
@@ -43,6 +50,15 @@ def textual_ext_management_system
     textual_link(ExtManagementSystem.find(@record.ems_id))
   end
 
+  def textual_connected_physical_servers
+    physical_servers_count = @record.physical_servers.count
+    physical_servers = {:label => _("Physical Servers"), :value => physical_servers_count, :icon => PhysicalServerDecorator.fonticon}
+    if physical_servers_count.positive?
+      physical_servers[:link] = "/physical_switch/show/#{@record.id}?display=physical_servers"
```

Comment:
I think you should try to use a helper method to calculate the URLs. I will help if we convert the controllers to restful ones. (The same on line 44). There's `show_link` in `GenericShowMixin` that you have already included in your controller. Seems to me you could use that one.

Reply:
Just realize this method: `url_for_only_path`. Do you think that I can use it?

Like:
```ruby
physical_servers[:link] = url_for_only_path(:action => 'show', :id => @record, :display => 'physical_servers')
```